widget/button.go

```diff
@@ -225,6 +205,13 @@ func (o ButtonOptions) TextAndImage(label string, face font.Face, image *ButtonI
 	}
 }
 
+func (o ButtonOptions) TextPosition(h TextPosition, v TextPosition) ButtonOpt {
```

Comment:
This looks great! Would you be willing to add comments to this function?

Reply:
Sure, done!